docs/upgrade_1-2_to_1-3.md

```diff
@@ -0,0 +1,81 @@
+# Upgrading from OADP 1.2
+
+> **NOTE:** Always upgrade to next minor version, do NOT skip versions. To update to higher version, do one upgrade at a time. Example: to upgrade from 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
+## Changes from OADP 1.2 to 1.3
+
+- Velero was updated from version 1.11 to 1.12 (Changes reference: https://velero.io/docs/v1.12/upgrade-to-1.12/#upgrade-from-v110-or-higher)
+
+    From this update, OADP now uses Velero Built-in DataMover instead of VSM/Volsync DataMover. This change the following:
+
+    - `spec.features.dataMover` was removed (you can delete secret for `spec.features.dataMover.credentialName` from your cluster, if you want)
+
+    - `vsm` plugin was removed
+
+    - `volsync` operator is not necessary anymore (you can uninstall it from your cluster, if you want)
+
+    - `volumesnapshotbackups.datamover.oadp.openshift.io` and `volumesnapshotrestores.datamover.oadp.openshift.io` CustomResourceDefinitions are not necessary anymore (you can delete `volumesnapshotbackups.datamover.oadp.openshift.io` and `volumesnapshotrestores.datamover.oadp.openshift.io` CRDs from your cluster, if you want)
+
+    Also, OADP now supports a new file system backup software: Kopia.
+
+    - To use it, use the new `spec.configuration.nodeAgent` field. Example
+
+        ```yaml
+        spec:
+          configuration:
+            nodeAgent:
+              enable: true
+              uploaderType: kopia
+        ```
+
+`spec.configuration.restic` field is being deprecated in OADP 1.3, and will be removed in OADP 1.4. To avoid seeing deprecating warnings about it, use the new syntax:
+```diff
+ spec:
+   configuration:
+-    restic:
+-      enable: true
++    nodeAgent:
++      enable: true
++      uploaderType: restic
+```
+
+> **Note:** OADP will be favoring Kopia over Restic in the near future.
+
+## Upgrade steps
+
+### Create new backup
+
+If you are using DataMover, you need to create backup before upgrade, because they 1.2 DataMover backups will not work after upgrade. We suggest to create Restic backups.
```

Comment:
I wasn't able to test that, still TODO

Reply:
Tested with mysql-persistent, and actually worked...

I ran csi, datamover and restic backups in stable-1.2 and restored with `make deploy-olm`, all """worked"""

All of them were **PartiallyFailed** with the same errors:

>mysql-persistent:  error getting resource from lister for rolebindings.authorization.openshift.io, mysql-persistent/system:deployers: rolebindings.authorization.openshift.io "system:deployers" not found
>                       error getting resource from lister for rolebindings.authorization.openshift.io, mysql-persistent/system:image-builders: rolebindings.authorization.openshift.io "system:image-builders" not found
>                       error getting resource from lister for rolebindings.authorization.openshift.io, mysql-persistent/system:image-pullers: rolebindings.authorization.openshift.io "system:image-pullers" not found

but when checking applications, they were running and were restored correctly. Can someone also run some backups to double check this behavior?

Reply:
we will test this in a specific day :)